### PR TITLE
U of T DSI | LCR - Assignment 1

### PR DESCRIPTION
**Pull request by Kristina-K96**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
I completed assignment 1.

## What did you learn from the changes you have made?
I learned how to create a knn model to classify a new observation.

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
I could have used the score() function instead of the accuracy_score() function to test the accuracy of the model.

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
I had some trouble using the train_test_split() function. I wasn't sure how to input the x and y variables so that they corresponded to the predictors and response values. I asked some of my peers for help with this (@eceyucer and @greergillies)!

## How were these changes tested?
I ran the code on my computer.

## A reference to a related issue in your repository (if applicable)
n/a 
## Checklist
- [x] I can confirm that my changes are working as intended

### CODE REVIEW

**Review by KasraVakiloroayaei — APPROVED** [review]
A1:Complete 

Good work

Q2part(ii) More Specifically, since it is a categorical variable, we do not standardize it
